sklearn/_loss/_loss.pyx.tp

```diff
@@ -901,7 +899,7 @@ else:
     with_param = ", self." + param
 }}
 
-cdef class {{name}}(CyLossFunction):
```

Comment:
Shouldn‘t it inherit from CyLossFunction?

Reply:
This was a mistake and I reverted it. (I was trying to figure out why the fused type of `cnp.float{32,64}` did not work).